src/main/webapp/js/grapheditor/Shapes.js

```diff
@@ -3658,10 +3658,9 @@ function getShape(cell) {
     } else if (style.shape === 'program') {
       graph.setCellStyles('name', value, [cell]);
       cell.setValue("Program: " + value);
-
-    } else {
-      cell.setValue(value);
+    } else if (style.shape === 'controlPlaneTableCell') {
       graph.setCellStyles('name', value, [cell]);
+      cell.setValue(value + "\n" +style.cType);
```

Comment:
```suggestion
      cell.setValue(`${value}\n${style.cType}`);
```

Just to make sure JS won't troll us with its messy typesystem.